Directory.Build.props

```diff
@@ -24,5 +24,6 @@
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>$(PackageProjectUrl).git</RepositoryUrl>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <LangVersion>7.3</LangVersion>
```

Comment:
This is redundant, we already use the latest version here: https://github.com/justeat/JustEat.StatsD/blob/a3ee3711594b048e94be4068fba54dffd6c63094/Directory.Build.props#L14